### ISSUE TITLE
Missing label on search field

### ISSUE THREAD

**Issue by zebdelk** [cedardevs/onestop#796]
Search Input Box is missing a label but has a title. This causes some screen readers to read "blank".

From MediaCurrent's A & AA findings.

- [x] add a label to Search Input Box for screen readers (make it invisible, since the design doesn't need a visual label)

**Comment by StrayCat1** [comment]
Branch -
https://github.com/cedardevs/onestop/tree/796-searchLabel
 
Commit - 
https://github.com/cedardevs/onestop/commit/090f2fc7737a5f65fb9ec44c46ac3693a629e690

I had a couple questions for @zebdelk -
1) I added the label to `<CollectionSearch>`. Is this the right place to add the label or did we want a more generic label in `<TextSearchField>`?
2) Is the text of the label adequate?
3) How can I test this?

**Comment by StrayCat1** [comment]
https://github.com/cedardevs/onestop/pull/902

**Comment by StrayCat1** [comment]
Setting up a VM for easy access testing.  This the pull request I have is still valid, but need to check for myself with a good screen reader.
